pkg/scheduler/api/job_info.go

```diff
@@ -166,6 +166,12 @@ func NewTaskInfo(pod *v1.Pod) *TaskInfo {
 		ti.Priority = *pod.Spec.Priority
 	}
 
+	if taskPriority, ok := pod.Labels["volcano.sh/task-priority"]; ok {
```

Comment:
Can we have a const for `volcano.sh/task-priority`?

And it's better to get this info from annotation instead of labels; AFAIK, we did several check against annotations, e.g. reject updating task priority :)

Reply:
ok, I have changed `volcano.sh/task-priority` from label to annotation and use `volcano.sh/task-priority` as a const